packages/test/test-end-to-end-tests/src/test/summarization/summarizeWithLocalChanges.spec.ts

```diff
@@ -68,655 +68,665 @@ async function waitForSummaryOp(container: IContainer): Promise<boolean> {
+				);
+				rootDataObject._root.set("store", dataObject.handle);
+				await waitForContainerConnection(mainContainer);
+
+				const summarySucceeded = await timeoutAwait(waitForSummaryOpPromise, {
+					errorMsg: "Timeout on waiting for summary op",
+				});
+				assert(summarySucceeded === true, "Summary should have been successful");
+
+				// The sequence of events that should happen:
+				// 1. First summarize attempt starts, i.e., summaryAttempts = 1.
+				// 2. Data store is created in summarizer.
+				// 3. Summarize cancels with NodeDidNotSummarize error.
+				// 4. Second summarize attempts starts, i.e., summaryAttempts = 2.
+				// 5. Summary is successfully generated.
+				const clientType = "noninteractive/summarizer";
+				const expectedEventsInSequence: Omit<ITelemetryBaseEvent, "category">[] = [
+					{
+						eventName: "fluid:telemetry:Summarizer:Running:Summarize_start",
+						clientType,
+						summaryAttempts: 1,
+					},
+					{
+						eventName: "fluid:telemetry:FluidDataStoreContext:DataStoreCreatedInSummarizer",
+						clientType,
+					},
+					{
+						eventName: "fluid:telemetry:Summarizer:Running:Summarize_cancel",
+						clientType,
+						error: "NodeDidNotSummarize",
+						summaryAttempts: 1,
+					},
+					{
+						eventName: "fluid:telemetry:Summarizer:Running:Summarize_start",
+						clientType,
+						summaryAttempts: 2,
+					},
+					{
+						eventName: "fluid:telemetry:Summarizer:Running:Summarize_generate",
+						clientType,
+						summaryAttempts: 2,
+					},
+				];
+
+				logger.assertMatch(expectedEventsInSequence, "Unexpected sequence of events");
 			},
-		],
-		async () => {
-			const logger = new MockLogger();
-			const mainContainer = await createContainer(
-				provider,
-				false /* disableSummary */,
-				logger,
-			);
-			const rootDataObject = (await mainContainer.getEntryPoint()) as RootTestDataObject;
-			const waitForSummaryOpPromise = waitForSummaryOp(mainContainer);
-			const dataObject = await dataStoreFactory1.createInstance(
-				rootDataObject.containerRuntime,
-			);
-			rootDataObject._root.set("store", dataObject.handle);
-			await waitForContainerConnection(mainContainer);
-
-			const summarySucceeded = await timeoutAwait(waitForSummaryOpPromise, {
-				errorMsg: "Timeout on waiting for summary op",
-			});
-			assert(summarySucceeded === true, "Summary should have been successful");
-
-			// The sequence of events that should happen:
-			// 1. First summarize attempt starts, i.e., summaryAttempts = 1.
-			// 2. Data store is created in summarizer.
-			// 3. Summarize cancels with NodeDidNotSummarize error.
-			// 4. Second summarize attempts starts, i.e., summaryAttempts = 2.
-			// 5. Summary is successfully generated.
-			const clientType = "noninteractive/summarizer";
-			const expectedEventsInSequence: Omit<ITelemetryBaseEvent, "category">[] = [
+			// The test is expected to take longer than the default timeout of 2s.
+		).timeout(standardTimeout * 2);
```

Comment:
For the record, I increased this to a hardcoded 5s instead of 2s locally and did not see this test fail in _many_ (1000+) iterations of the e2e tests against local server. This is more or less equivalent, I like it.